keyboards/momoka_ergo/keymaps/via/rules.mk

```diff
@@ -0,0 +1,9 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+LEADER_ENABLE = no
+FAUXCLICKY_ENABLE = no
+MIDI_ENABLE = no
+BLUETOOTH_ENABLE = no
+KEY_LOCK_ENABLE = no
+TERMINAL_ENABLE = no
+BOOTMAGIC_ENABLE = lite
```

Comment:
```suggestion
```

Reply:
@fauxpark Just so I understand, this isn't necessary?
I was following the recommendation here: https://www.caniusevia.com/docs/configuring_qmk/#create-a-rulesmk-in-keyboardskeyboard_namekeymapsvia

Reply:
That documentation is outdated; those features no longer change the keycode values. However it's not necessary to add them anyway if the keyboard-level rules.mk does not set them to `yes`, as they effectively default to `no`.

Bootmagic Full has been removed, and Lite is now the default, so `lite` is not a valid value here. Additionally, `VIA_ENABLE = yes` implies `BOOTMAGIC_ENABLE = yes` so there is no need to explicitly enable it.